src/amber/controller/helpers/responders.cr

```diff
@@ -61,10 +62,11 @@ module Amber::Controller::Helpers
 
     private def requested_responses
       req_responses = Array(String).new
-      if (ext = request.path.match(Content::REGEX).try(&.[1]))
-        req_responses << Content::TYPE[ext]
+      path_ext = request.path.match(Content::REGEX).try(&.[1])
```

Comment:
Non blocking comment: This would probably read better in a private method called format.

```crystal
def requested_format
  request.path.match(Content::REGEX).try(&.[1])
end
```

Reply:
Possibly but it's only one source of format. The private method that it's in is already tasks with finding the requested response types.